### PR TITLE
dev-python/qscintilla-python: add Qt6 support

### PR DESCRIPTION
**Pull request by t0b3**

### CODE REVIEW

File: dev-python/qscintilla-python/qscintilla-python-2.13.3.ebuild [hunk 1]
```diff
@@ -21,60 +21,78 @@ S=${WORKDIR}/${MY_P}/Python
...
+			"${myconf[@]}" || die
 
-		# Run eqmake to respect toolchain and build flags
-		run_in_build_dir eqmake5 -recursive ${MY_PN}.pro
+			# Run eqmake to respect toolchain and build flags
+			run_in_build_dir "${QMAKE}" -recursive ${MY_PN}.pro
+		}
+		case ${MULTIBUILD_VARIANT} in
+			qt5) local QMAKE=eqmake5 ;;
+			qt6) local QMAKE=qmake6 ;;
```
Comment (AndrewAmmerlaan) [inline dev-python/qscintilla-python/qscintilla-python-2.13.3.ebuild:80]: Same here. Perhaps we should wait till we have a `eqmake6` function.

Comment (t0b3) [reply dev-python/qscintilla-python/qscintilla-python-2.13.3.ebuild:80]: sure, let's wait for https://github.com/gentoo/gentoo/pull/31254 to include `eqmake6` ...

**Comment by Pesa** [comment]
I'd recommend folding this PR into #31252 (keeping separate commits of course)

**Comment by t0b3** [comment]
folded into #31252 :+1:

**Pull request closed without merging**